src/js/consts/timeFormat.js

```diff
@@ -4,5 +4,5 @@ export const TimeFormat = {
 
 export const DateFormat = {
   MM_DD_YYYY: 'MM/DD/YYYY',
-  MMM_DD_YYYY: 'MMM DD, yyyy',
+  LL: 'll',
```

Comment:
What's wrong with the previous version? If you'd like to add `LL` (equivalent of `MMMM DD, yyyy`) which is also a valid case how would the key look like?

Reply:
I don't have any suggestion at the moment, I would assume that we won't use the "LL" in the near future :thinking: or just go with a more intuitive name (I haven't known what LL is tbh), so let's maybe go with convention of naming keys what they really display?